database/controllers.js

```diff
@@ -18,6 +18,22 @@ const getUser = () => {
     .catch(errorHandler);
 };
 
+const getLeaderboard = () => {
+  const queryString = 'SELECT * FROM users ORDER BY winnings DESC LIMIT 25';
```

Comment:
probably don't want to select passwords here when sending the leaderboard down to the client